boards/arm/steval_fcu001v1/steval_fcu001v1.dts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f401Xe.dtsi>
```

Comment:
Board is using STM32F401CC (with 64Kb RAM, 256K Flash), you need to add the matching stm32f401Xc.dtsi file (stm32f401Xe.dtsi declares 96Kb RAM, 512K Flash).